index.js

```diff
@@ -26,6 +26,8 @@ const ServerResponse = require('fwsp-server-response');
 let serverResponse = new ServerResponse();
 const UMFMessage = require('fwsp-umf-message');
 
+const RedisConnection = require('./redis');
```

Comment:
./redis.js should be renamed to avoid visual conflict with the standard redis import:
https://github.com/flywheelsports/fwsp-hydra/blob/master/index.js#L15